pkgs/swiftgen/swift2objc/analysis_options.yaml

```diff
@@ -2,23 +2,6 @@ include: package:dart_flutter_team_lints/analysis_options.yaml
 
 analyzer:
   language:
-    strict-casts: false # TODO(https://github.com/dart-lang/native/issues/1278): Fix.
-    strict-inference: false # TODO(https://github.com/dart-lang/native/issues/1278): Fix.
-    strict-raw-types: false # TODO(https://github.com/dart-lang/native/issues/1278): Fix.
 
 linter:
```

Comment:
Since the `rules` are now empty, you can delete the `linter` key entirely.

Reply:
done